cli/src/commands/git/init.rs

```diff
@@ -183,7 +184,13 @@ fn do_init(
                     tx.finish(ui, "import git head")?;
                 }
             }
-            print_trackable_remote_bookmarks(ui, workspace_command.repo().view())?;
+            if settings.get_bool("git.track-on-init")? {
+                let mut checkout_tx = workspace_command.start_transaction();
```

Comment:
Find a better name :)

Actually, maybe we should just use the same transaction as for the import above? Should be able to move the `let mut tx = ...` line outside the block and also move the `if tx.repo().has_changes()` block later.